assignments/arrays.js

```diff
@@ -63,7 +63,8 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(inventory.id[32], );
```

Comment:
console.log(inventory.id[32], ) will log the ENTIRE car 33 object.. if you'd like to log a string, instead, with the requested values interpolated(inserted), you could use back ticks instead of quotes to signify a string and using the ${} syntax, insert the values you'd like!
example:

console.log(`Car 33 is a ${inventory.id[32].car_year} ${inventory.id[32].car_make} ${inventory.id[32].car_model}` );